Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -391,7 +353,9 @@ def bump_formula_pr
             #{user_message}
           EOS
         end
-        safe_system "hub", "pull-request", *hub_args, "-m", pr_message
+        pr_title = "#{formula.name} #{new_formula_version}#{devel_message}"
+        GitHub.create_pull_request(formula.tap.full_name, pr_title,
```

Comment:
Even this can fail. How should the exception be handled here ?

Reply:
The backup file needs to be restored with

```
formula.path.atomic_write(backup_file) unless ARGV.dry_run?
```
or equivalent